content/source/docs/cloud/api/ip-ranges.html.md

```diff
@@ -24,14 +24,15 @@ page_title: "IP Ranges - API Docs - Terraform Cloud and Terraform Enterprise"
 [CIDR Notation]: https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation
 
 # IP Ranges API
-IP Ranges provides a list of Terraform Cloud and Enterprise's egress IP ranges. For more information about Terraform Cloud's IP ranges, view our documentation about [Terraform Cloud IP Ranges](../architectural-details/ip-ranges.html).
+IP Ranges provides a list of Terraform Cloud and Enterprise's IP ranges. For more information about Terraform Cloud's IP ranges, view our documentation about [Terraform Cloud IP Ranges](../architectural-details/ip-ranges.html).
 
 ## IP Ranges Payload
 
 Name                             | Type   | Description
 ---------------------------------|--------|-------------
-`sentinel`                       | array  | List of IP ranges in [CIDR notation] used for outbound requests from Sentinel policies
+`api`                            | array  | List of IP ranges in [CIDR notation] used for Terraform Cloud APIs
```

Comment:
These are the IP addresses which a customer must allow `outbound` from their local site to communicate with TFC, right? If so, propose the following change:

```suggestion
`api`                            | array  | List of IP ranges in [CIDR notation] used for connections from user site to Terraform Cloud APIs
```

Reply:
yes perfect 👍 that sounds better. thanks @vaficionado